### DISCUSSION TITLE
"State of the Headless CMS 2018"

### DISCUSSION THREAD

**Discussion by tarciozemel** [frontendbr/forum#1160] [Discussão]
https://kenticocloud.com/state-of-headless-cms-2018

PS: Não cometam o mesmo erro da pesquisa do Stack Overflow. Os números refletem as repostas _dos que participaram da pesquisa_, não de tudo o que acontece na via láctea sobre o assunto.

**Comment by rubenmarcus** [comment]
Se voce hoje fosse fazer um one page em wordpress, voce faria headless, ou um tema normal? @tarciozemel  ?

**Comment by tarciozemel** [comment]
@rubenmarcus Depende dos requisitos do projeto.
